api/v1alpha1/ratelimit_types.go

```diff
@@ -62,6 +64,11 @@ type RateLimitList struct {
 	Items           []RateLimit `json:"items"`
 }
 
+type NamespacedName struct {
```

Comment:
it exists in apimachinery https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName